### ISSUE TITLE
Auto import of package/class when a candidate is selected doesn't work for Java files/projects.

### ISSUE THREAD

**Issue by biplabswain** [manateelazycat/lsp-bridge#1032]
Hi,

Auto import of package/class when a candidate is selected doesn't work for Java files/projects. The package/class information a candidate belongs to is displayed in the completion menu but it doesn't get inserted into the buffer upon selection of the candidate. Auto import works as expected for C files/projects. Auto import works for Java when I use lsp-mode for the same Java project and the same installed Java language server(jdtls) so I guess this is a configuration issue?. I compared the lsp-bridge log for Java and C and noticed that in case of C, header file information for auto import is returned in additionalTextEdits, but class information for auto import is not returned in additionalTextEdits for Java.

I'm using Emacs 29.4 running on Fedora Linux 40 (Workstation Edition). Linux 6.10.6-200.fc40.x86_64. lsp-bridge version - pulled as of today.

Jdtls version - https://www.eclipse.org/downloads/download.php?file=/jdtls/milestones/1.39.0/jdt-language-server-1.39.0-202408291433.tar.gz

Thank you.

**Comment by manateelazycat** [comment]
@zsxh @vritser Can you help this issue? Thank you . ;)

**Comment by biplabswain** [comment]
Hi,

I did further testing and found out that lsp-bridge does actually auto import of package/class but only when acm-enable-doc is set to true. Auto import doesn't work when acm-enable-doc is set to nil. Before I opened this issue I had set acm-enable-doc to nil, otherwise, for certain cases, the document popup is displayed over acm popup.

Following is the minimal configuration to reproduce the issue.

```
(add-to-list 'load-path (expand-file-name "~/path/to/lsp-bridge"))

(use-package lsp-bridge
  :custom
  (acm-enable-doc nil)
  :config
  (global-lsp-bridge-mode))
```

Thanks.
